### PR TITLE
Account notifications refactor

### PR DESCRIPTION
**Pull request by alioso**
This PR places the account notifications in its own dropdown to avoid intrusive & permanents alerts within the app frame.

### CODE REVIEW

**Comment by alioso** [comment]
@rjstires back to you for help with logic. We need:

1. make sure we display the button IF we have any notification of this type. I did not find an efficient way to do that so far. I think this will have to be done with a reducer which has to be done with #2 in this list
2. there is an ``isImportant`` prop which we need to set to true if any notification is high priority. This will set the button color to red.

To Do: implement pattern in storybook

**Comment by rjstires** [comment]
Rebased onto `release-0.25.1` and updated per your request. Looks and feels great. This is the way it should have been since the start.

File: e2e/config/wdio.conf.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ const password = process.env.MANAGER_PASS;
 
 const specsToRun = () => {
     if (argv.file) {
-        return [argv.file];
+        return ['./e2e/setup/setup.spec.js', argv.file];
```
Comment (patthiel) [inline e2e/config/wdio.conf.js]: this should already be in develop

File: e2e/config/wdio.conf.js [hunk 2]
```diff
@@ -295,7 +295,6 @@ exports.config = {
             })
             .catch(err => {
                 console.log(err);
-                reject(err);
```
Comment (patthiel) [inline e2e/config/wdio.conf.js]: and this

**Comment by patthiel** [comment]
Seems a bunch of my changes already in develop have made there way into this PR. Also see there are merge conflicts. I'll take a look at this and test once all is resolved

**Comment by rjstires** [comment]
@patthiel the diff is just junked because of the merge conflicts. I'll all wash out.

**Comment by rjstires** [comment]
Closing, please refer to 3374.

**Pull request closed without merging**